src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/BestemmendeFravaersdag.kt

```diff
@@ -9,15 +9,18 @@ fun bestemmendeFravaersdag(
     egenmeldingsperioder: List<Periode>,
     sykmeldingsperioder: List<Periode>,
 ): LocalDate {
-    val sisteArbeidsgiverperiode = arbeidsgiverperioder.ifEmpty { null }
-        ?.sisteSammenhengendePeriode { denne, neste ->
+    val sisteArbeidsgiverperiode = arbeidsgiverperioder
+        .slaaSammenSammenhengendePerioder { denne, neste ->
             denne.tom.daysUntil(neste.fom) <= 1
         }
+        .lastOrNull()
 
     val sisteSykdomsperiode = (egenmeldingsperioder + sykmeldingsperioder)
```

Comment:
litt usikker, men er ikke egentlig dette siste delperioden innen 16 dager grensen / agp.
Så ville det vært mer beskrivende å kalle varabelen sisteSykdomsperiodeInnenAgp feks?

Reply:
Hva med `sistePeriodeIAgp`?
Siste periode "innen" høres ut som en periode som delvis er inkludert i AGP.

Reply:
Nei, dette er siste periode i hele sykdomsfraværet, uavhenging av AGP.

Reply:
Da tror jeg ikke jeg kjønner hva den filtrerBortUtgaaendePerioder gjør.
Filtrerer ikke den bort det som overstiger 16 dager?

Reply:
Den funksjonen fikk nettopp det nye navnet `fjernPerioderEtterFoersteUtoverAgp`. Så den fjerner alle perioder etter den første perioden som er utenfor AGP / 16 dager. Men hvis du har to perioder, første på 16 dager og andre på 12 dager, så brukes `fom` fra andre periode. Andre periode er i dette tilfellet frakoblet AGP.